migrations/3_deploy_adminwallet.js

```diff
@@ -0,0 +1,58 @@
+const AdminWallet = artifacts.require("./AdminWallet.sol");
+
+// if (process.env.NODE_ENV !== 'production') { // https://codeburst.io/process-env-what-it-is-and-why-when-how-to-use-it-effectively-505d0b2831e7
+require("dotenv").load();
+// }
+
+const PrivateKeyProvider = require("truffle-hdwallet-provider-privkey");
+const HDWalletProvider = require("truffle-hdwallet-provider");
+
+const admin_mnemonic = process.env.ADMIN_MNEMONIC;
+const infura_api = process.env.INFURA_API;
+
+module.exports = async function(deployer, network, provider) {
```

Comment:
i think there is not provider parameter, its accounts i think

Reply:
Neither are needed